src/main/resources/assets/tconstruct/lang/zh_cn.json

```
@@ -0,0 +1,2583 @@
+  "modifier.tconstruct.cooling.attack_damage": "冷却伤害",
+  "modifier.tconstruct.sweeping_edge": "横扫",
+  "modifier.tconstruct.sweeping_edge.flavor": "武士！",
+  "modifier.tconstruct.sweeping_edge.description": "横扫攻击的目标在此工具的伤害中占较大比例",
+  "modifier.tconstruct.sweeping_edge.attack_damage": "横扫伤害",
+
+  "modifier.tconstruct.sharpness": "锋利",
+  "modifier.tconstruct.sharpness.1": "锋利+",
+  "modifier.tconstruct.sharpness.2": "锋利++",
+  "modifier.tconstruct.sharpness.3": "锋利+++",
+  "modifier.tconstruct.sharpness.4": "锋利++++",
+  "modifier.tconstruct.sharpness.5": "锋利++++",
+  "modifier.tconstruct.sharpness.flavor": "有多锋利？",
+  "modifier.tconstruct.sharpness.description": "工具更加锋利,造成更多伤害！",
+  "modifier.tconstruct.swiftstrike": "迅击",
+  "modifier.tconstruct.swiftstrike.1": "迅击+",
+  "modifier.tconstruct.swiftstrike.2": "迅击++",
+  "modifier.tconstruct.swiftstrike.3": "迅击+++",
+  "modifier.tconstruct.swiftstrike.4": "迅击++++",
+  "modifier.tconstruct.swiftstrike.5": "迅击+++++",
+  "modifier.tconstruct.swiftstrike.flavor": "魔法！",
+  "modifier.tconstruct.swiftstrike.description": "神奇的紫色宝石让武器更轻更容易挥舞！",
+
+  "_comment": "Armor",
+  "modifier.tconstruct.knockback_resistance": "抗击性",
+  "modifier.tconstruct.knockback_resistance.flavor": "一个有分量的主题",
+  "modifier.tconstruct.knockback_resistance.description": "铁砧很重,所以它应该可以防止你被击退,对吧？",
+  "modifier.tconstruct.golden": "金闪闪",
+  "modifier.tconstruct.golden.flavor": "好如金子",
+  "modifier.tconstruct.golden.description": "给你的盔甲额外的金光闪闪,猪灵喜欢它！",
+  "modifier.tconstruct.ricochet": "弹性",
+  "modifier.tconstruct.ricochet.flavor": "额外的弹性",
+  "modifier.tconstruct.ricochet.description": "受到攻击时弹开更远,有效逃脱",
+  "modifier.tconstruct.revitalizing": "生命值加强",
+  "modifier.tconstruct.revitalizing.flavor": "能感觉到能量流动",
+  "modifier.tconstruct.revitalizing.description": "强化心,给予更多心",
+
+  "modifier.tconstruct.embellishment": "点缀",
+  "modifier.tconstruct.embellishment.formatted": "%s 装饰",
+  "modifier.tconstruct.embellishment.flavor": "重新装修！",
+  "modifier.tconstruct.embellishment.description": "装甲外观改变以匹配不同的材料",
+  "modifier.tconstruct.dyed": "染色",
+  "modifier.tconstruct.dyed.formatted": "染色 #%s",
+  "modifier.tconstruct.dyed.flavor": "重新装修！",
+  "modifier.tconstruct.dyed.description": "装甲外观改变以匹配不同的材料",
+
+  "_comment": "Armor - Protection",
+  "modifier.tconstruct.protection": "保护",
+  "modifier.tconstruct.protection.flavor": "万事通！",
+  "modifier.tconstruct.protection.description": "增加对所有伤害类型的保护",
+  "modifier.tconstruct.protection.resistance": "抵抗力",
+  "modifier.tconstruct.melee_protection": "近战保护",
+  "modifier.tconstruct.melee_protection.flavor": "砰！",
+  "modifier.tconstruct.melee_protection.description": "增加对直接物理伤害的保护",
+  "modifier.tconstruct.melee_protection.resistance": "近战抗性",
+  "modifier.tconstruct.fire_protection": "火焰保护",
+  "modifier.tconstruct.fire_protection.flavor": "变成冶炼炉！",
+  "modifier.tconstruct.fire_protection.description": "保护免受火灾伤害并减少火灾时间",
+  "modifier.tconstruct.fire_protection.resistance": "耐火性",
```

```suggestion
  "modifier.tconstruct.fire_protection.resistance": "火焰抗性",
```
xx resistence 格式保持一致可能会更好。